index.js

```diff
@@ -0,0 +1,236 @@
+    console.log(`THE GAMEMODE IS NOW: ${gameMode}`)
+}
+
+//function to update clock
+function updateClock() {
+    clockCount++
+    clock.innerHTML = `Time Elapse: ${clockCount} seconds`
+}
+
+//function to change status
+function showPlayer() {
+    if (currentPlayer === 1) {
+        gameStatus.innerHTML = `Player ${playerOneDisplay}'s turn`
+    } else {
+        gameStatus.innerHTML = `Player ${playerTwoDisplay}'s turn`
+    }
+}
+
+// function to prevent clicking on the same cell
+function alreadyClicked() {
+    gameStatus.innerHTML = `Please select an empty cell.`
+}
+
+//function to check for wins
+function checkWin(currentPlayerArr) {
+
+    let winner = false
+    wins.forEach((win) => {
+        if (currentPlayerArr.includes(win[0])) {
+            if (currentPlayerArr.includes(win[1])) {
+                if (currentPlayerArr.includes(win[2])) {
+                    winCom.push(...win)
+                    console.log(winCom)
+                    console.log("You Won!")
+                    winner = true
+                }
+            }
+        }
+    })
+    return winner
+}
+
+//prevents from clicking after win
+function reStart() {
+    for (let cell of cells) {
+        cell.removeEventListener('click', clicked)
+        cell.removeEventListener('click', alreadyClicked)
+    }
+    startButton.disabled = false
+    playerOneName.disabled = false
+    playerTwoName.disabled = false
+    playerOneName.value = ""
+    playerTwoName.value = ""
+}
+
+//makeMove function - when player clicks on cells
+function clicked(event) {
+    //movesMade += 1 - not using this variable right now
+    event.target.removeEventListener('click', clicked)
+    event.target.addEventListener('click', alreadyClicked)
+    let cellNumber = parseInt(event.target.id)
+    let index = boardArr.indexOf(cellNumber)
+
+
+    if (boardArr.includes(cellNumber)) {
+        console.log(boardArr)
+        boardArr.splice(index, 1)
+    }
+    if (currentPlayer === 1) {
+        event.target.innerHTML = playerOne
+        event.target.style.color = "red"
+        currentPlayer += 1
+        playerOneArr.push(cellNumber)
+        if (checkWin(playerOneArr) === true) {
+            gameStatus.innerHTML = `Congratulations ${playerOneDisplay}! You won!`
+            clearInterval(interval)
+
+            for (let cell of cells) {
+                console.log(cell.id)
+                if (winCom.includes(+cell.id))
+                    cell.style.backgroundColor = "green"
+            }
+            return reStart()
+        }
+        
+        if (boardArr.length === 0) {
+            gameStatus.innerHTML = `It's a draw`
+            clearInterval(interval)
+            startButton.disabled = false
+        }
+        
+        if(gameMode === 'pvc'){
+            randoCellNum = randomNum(0, boardArr.length-1).toString()
+            randoCell = document.getElementById(boardArr[randoCellNum])
+            randoCell.click()
+            currentPlayer = 1
+        }
```

Comment:
Awesome job integrating the computer player!  Once you realize that all you really need to do is simulate a click, I feel like things make a lot more sense. Awesome way of creating a random cell, too. You avoid the issues with recursive computer guesses a lot of people usually encounter by doing it this way.  Nicely done.